packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1610,6 +1894,8 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
     final ScrollMetrics metrics = notification.metrics;
     if (_shouldUpdatePainter(metrics.axis)) {
       scrollbarPainter.update(metrics, metrics.axisDirection);
+      _lastMetrics = metrics;
```

Comment:
If the thumb reaches the edge of the track, the `Scrollbar` button may need to update the indicator styles.